src/components/Comments.jsx

```diff
@@ -6,26 +6,36 @@ import DeleteComment from "./DeleteComment.jsx";
 export default function Comments ({article_id}) {
 
     const [comments, setComments] = useState([])
-    const [author, setAuthor] = useState(null)
+    const [postedComment, setPostedComment] = useState({})
+    const [err, setErr] = useState(null)
+    
 
     useEffect(() => {
 
         fetchCommentsById(article_id).then(({data}) => {
             setComments(data.comments)
         }).catch((err) => {
+            setErr(err.message)
             console.log(err)
         })
 
     },[comments])
 
+    if (err) {
+        return <div>
+            <h3>Sorry 😬 could not load comments...</h3>
+            <p>{err}</p>
+        </div>
+    }
+
     return (
         <div id="comments">
             <h3>Comments:</h3>
-            <PostComment article_id={article_id} setAuthor={setAuthor}></PostComment>
+            <PostComment article_id={article_id} setPostedComment={setPostedComment}></PostComment>
             <ul>
                 {comments.map((comment) => {
                     return <li key={comment.comment_id}>
-                        {comment.author}: {comment.body} { comment.author === author?<DeleteComment comment_id={comment.comment_id}></DeleteComment> : null}
+                        {comment.author}: {comment.body} { comment.author === postedComment.author?<DeleteComment comment_id={comment.comment_id}></DeleteComment> : null}
```

Comment:
At the moment the delete button only appears once the user has posted a comment, can you have it so that the delete button is there as soon as the page renders?